src/components/header-checkout/header-checkout.hbs

```diff
@@ -0,0 +1,181 @@
+          Boys
+          <svg class="icon icon--chevron">
+            <use xlink:href="{{rootPath}}/sprite.svg#chevron"></use>
+          </svg>
+        </button>
+        <ul class="primary-nav-checkout__submenu">
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Boys</a>
+          </li>
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Tops</a>
+          </li>
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Pants</a>
+          </li>
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Dresses</a>
+          </li>
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Shoes</a>
+          </li>
+          <li class="submenu-item-checkout">
+            <a href="{{rootPath}}/plp/index.html">Hats</a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </nav>
+  <nav class="secondary-nav-checkout" aria-label="Secondary" data-js="hamburger-dropdown">
+    <ul class="secondary-nav__items">
+      <li class="secondary-nav__item">
+        <button class="secondary-nav__button" data-modal-show="construction-modal">
+          <svg class="icon icon--menu-btn">
+            <use xlink:href="{{rootPath}}/sprite.svg#account"></use>
+          </svg> Account
+        </button>
+      </li>
+      <li class="secondary-nav__item secondary-nav__mini-cart mini-cart__toggle" data-component="mini-cart">
+        <button aria-label="Mini Cart, 2 items, estimated subtotal: $42.92" data-js="mini-cart-toggle" class="secondary-nav__button" aria-haspopup="true" aria-expanded="false">
+          <span aria-live="assertive" aria-atomic="true" class="element-invisible mini-cart__aria">You have 0 items in
+            your cart.</span>
+          <svg class="icon icon--menu-btn">
+            <use xlink:href="{{rootPath}}/sprite.svg#cart"></use>
+          </svg>
+          Cart
+          <span class="mini-cart__visible-label--amount" data-js="cart-item-count">0</span>
+        </button>
+        <div data-template="mini-cart"></div>
+      </li>
+      <li class="secondary-nav__item secondary-nav__stores">
+        <button class="secondary-nav__button" data-modal-show="construction-modal">
+          <svg class="icon icon--menu-btn">
+            <use xlink:href="{{rootPath}}/sprite.svg#stores"></use>
+          </svg> Stores
+        </button>
+      </li>
+      <li class="secondary-nav__item secondary-nav__contact">
+        <button class="secondary-nav__button" data-modal-show="construction-modal">
+          <svg class="icon icon--menu-btn">
+            <use xlink:href="{{rootPath}}/sprite.svg#contact"></use>
+          </svg> Contact
+        </button>
+      </li>
+    </ul>
+  </nav>
+  <div class="checkout-info">
+    <h2 class="checkout-info__heading">
```

Comment:
I used a `<h2>` for this. Let me know if this should be a different heading tag.